ambry-store/src/main/java/com.github.ambry.store/IndexValue.java

```diff
@@ -13,114 +13,326 @@
  */
 package com.github.ambry.store;
 
+import com.github.ambry.utils.Time;
 import com.github.ambry.utils.Utils;
 import java.nio.ByteBuffer;
 
 
 /**
  * Represents the blob value stored in the index for a key.
+ *
+ * Version_0
+ * - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+ * | Blob Size |   Offset  |  Flags  | Expiration Time | Orig msg  |
+ * | (8 bytes) | (8 bytes) | (1 byte)|     in Ms       | offset    |
+ * |           |           |         |     ( 8 bytes)  | (8 bytes) |
+ * - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+ *
+ * Version_1
+ * - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+ * | Blob Size |   Offset  |  Flags  | Expiration Time | Orig msg  | OperationTime | ServiceId | ContainerId |
```

Comment:
We can't rely on the fact that the index key will always support these. We can potentially remove them from the index key and store just the UUID or may be with partition. Just that we don't want to stick to that assumption